docs/en/guide/compile/how-to-compile.md

```diff
@@ -0,0 +1,19 @@
+# Compiling
+
+## Go version
+
+Go version `1.17` or higher is supported for compilation.
+
+## Platform
+Linux is supported in the SkyWalking Rover.
```

Comment:
Linux is not supported out of box, you need to specify

- Minimum version of Linux kernel
- What dependencies they need to install `apt install ...`

Reply:
Updated, please recheck.